app/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -987,7 +987,7 @@ msgstr "Os bydd y broblem hon yn parhau, <a href='{url}'>cysylltwch â ni </a>i
 
 #: templates/individual_response/confirmation-post.html:15
 msgid "A letter has been sent to Individual Resident at {display_address}"
-msgstr "Mae llythyr wedi cael ei anfon at Ymatebwr Unigol yn {display_address}"
+msgstr "Mae llythyr wedi cael ei anfon at Preswylydd Unigol yn {display_address}"
```

Comment:
Again won't make a change request here as this has come from crowdin, but I beleve that there should be a soft mutation after `at` and it should be `Mae llythyr wedi cael ei anfon at Breswylydd Unigol yn {display_address}`

Reply:
Done now @berroar, thanks!